### PR TITLE
Randomized fetch hook

### PR DESCRIPTION
**Pull request by SharathRaparthy**

### CODE REVIEW

File: randomizer/config/ResidualFetchHookRandomized/random.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "env": "FetchPushDefault-v0",
```
Comment (bhairavmehta95) [inline randomizer/config/ResidualFetchHookRandomized/random.json:2]: Change the name to be correct for all environments.

File: randomizer/residual_envs/residual_fetch_pick_and_place_env.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+from gym import error, spaces, utils
+from gym.utils import seeding
+
+import numpy as np
+import time
+
+import pdb
+
+
+class MyFetchPickAndPlaceEnv(gym.Env):
```
Comment (bhairavmehta95) [inline randomizer/residual_envs/residual_fetch_pick_and_place_env.py:14]: why is this the name?

**Review by bhairavmehta95 — CHANGES_REQUESTED** [review]

File: randomizer/config/ResidualFetchHookRandomized/random.json (hunk 1 above)
Comment (SharathRaparthy) [reply randomizer/config/ResidualFetchHookRandomized/random.json:2]: okay! I did.

File: randomizer/residual_envs/residual_fetch_pick_and_place_env.py (hunk 2 above)
Comment (SharathRaparthy) [reply randomizer/residual_envs/residual_fetch_pick_and_place_env.py:14]: That's what they named in rpl_environments. I didn't change any of their stuff

**Pull request merged**